<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HashWidget</class>
 <widget class="QWidget" name="HashWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>861</width>
    <height>710</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Hash</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="placeholderText">
        <string>Check for integrity</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pasteIntegrityCheckButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::EditPaste"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearIntegrityCheckButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::EditClear"/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Input</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="openInputButton">
           <property name="toolTip">
            <string>Open</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset theme="QIcon::ThemeIcon::DocumentOpen"/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="copyInputButton">
           <property name="toolTip">
            <string>Copy</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset theme="QIcon::ThemeIcon::EditCopy"/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pasteInputButton">
           <property name="toolTip">
            <string>Paste</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset theme="QIcon::ThemeIcon::EditPaste"/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="clearInpututton">
           <property name="toolTip">
            <string>Clear</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset theme="QIcon::ThemeIcon::EditClear"/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="input"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Output</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="copyOutputButton">
           <property name="toolTip">
            <string>Copy</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset theme="QIcon::ThemeIcon::EditCopy"/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="output">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
